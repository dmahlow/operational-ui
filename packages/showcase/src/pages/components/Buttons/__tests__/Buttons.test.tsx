import * as React from "react"
import { render } from "enzyme"

import ButtonsPage from "../Buttons"

describe("Buttons Page", () => {
  it("Should render correctly", () => {
    expect(render(<ButtonsPage />)).toMatchSnapshot()
  })
})
