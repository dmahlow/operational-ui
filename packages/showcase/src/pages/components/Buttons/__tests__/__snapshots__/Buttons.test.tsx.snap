// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Buttons Page Should render correctly 1`] = `
<div>
  <div
    class="css-by30gj"
  >
    Buttons
  </div>
  <p>
    Buttons clearly indicate a point of interaction that allow users to perform actions in applications. On Contiamo, buttons exist independently, and in groups. These buttons can also take on any number of colors required.
  </p>
  <h4>
    Usage
  </h4>
  <div
    class="playground"
  >
    <div
      class="playgroundCode"
    >
      <div
        class="ReactCodeMirror playgroundStage"
      >
        <textarea
          autocomplete="off"
        >
          t]
        </textarea>
      </div>
    </div>
    <div
      class="playgroundPreview"
    >
      <div>
        <div
          class="previewArea"
        />
      </div>
    </div>
  </div>
  <h4>
    Props
  </h4>
  <table
    class="css-81lp1j"
  >
    <thead>
      <tr>
        <th>
          Name
        </th>
        <th>
          Description
        </th>
        <th>
          Default
        </th>
        <th>
          Type
        </th>
        <th>
          Optional?
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>
          color
        </td>
        <td>
          What color of button would you like? It can be a hex value or a named theme color.
        </td>
        <td>
          white
        </td>
        <td>
          string
        </td>
        <td>
          ✅
        </td>
      </tr>
      <tr>
        <td>
          onClick
        </td>
        <td>
          What happens when the button is clicked?
        </td>
        <td />
        <td>
          func
        </td>
        <td>
          ✅
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
